README.md

```diff
@@ -116,18 +116,59 @@ The tool to be used for formatting files can be configured with the following se
 "rubyLsp.formatter": "auto"
 ```
 
+#### Ruby version requirement
+
+By default, the Ruby LSP always uses the current project's Ruby version and bundle. This allows the LSP to find the
+correct gem versions automatically to index and to format the code matching behavior on CI.
+
+The Ruby LSP and its main dependency Prism (the new Ruby parser) both follow the same policy, which is to support only
+Ruby versions that are not beyond their end-of-life.
+
+If you're working on a project with an older Ruby version, it might be possible to install older versions of the server
+gem to get support for older rubies, but that might also involve using older versions of the VS Code extension - since
+some functionality requires implementations in both client and server.
+
+The other alternative is to use a custom Gemfile separate from the project with a different Ruby version. Notice that
+certain functionality may be degraded or require manual configuration, since the Ruby LSP will not be able to inspect
+the project's real bundle to discover dependencies. Please, see the instructions below.
+
 #### Using a custom Gemfile
 
 If you are working on a project using an older version of Ruby not supported by Ruby LSP, then you may specify a
-separate `Gemfile` for development tools. You can include the `ruby-lsp` in it and point to that Gemfile by using the
-following configuration:
+separate `Gemfile` for development tools.
 
 **Note**: when using this, gems will not be installed automatically and neither will `ruby-lsp` upgrades.
 
+Create a folder to store the custom bundle outside of the project that uses the old Ruby version. Inside that folder,
```

Comment:
(nit) we should really say `directory` rather than `folder`, since folder is a GUI abstraction.